rosette/src/Interrupt.cc

```diff
@@ -102,7 +102,6 @@ void RosetteIoHandler(VM_EVENT type, int fd, void*) {
 #endif
 
     if (type == VM_READ_EVENT) {
-        DO_UNGETC;
```

Comment:
I'm not sure what this did, but if the system works without it, I approve.

Reply:
All it did was call `getc` and then `ungetc`, which used to be a common hack to deal with some bugs in various Unix I/O systems, but is definitely unnecessary on modern Linux.

Reply:
Ahh.  Good to get rid of it then.  I've just never seen this particular idiom before.